internal/server/handler_v1.go

```diff
@@ -183,38 +183,78 @@ func (s *Server) VariableInfo(
 	return info.VariableInfo(ctx, in, s.store)
 }
 
+var storeBulkVariableInfo = info.BulkVariableInfo
```

Comment:
nit: bulkVariableInfoLocalFunc? Also make this a naming convention for all these stub functions?

Use local to contrast with remote and be consistent with "localResp" below

Reply:
Done.